test/python/test_cc_decoder.py

```diff
@@ -73,38 +81,22 @@ def test_maxsat_decoder(code: cc_decoder.HexagonalColorCode) -> None:
     assert est[1] == 1
 
 
-def test_simulate(distance: int, p: float, nr_sims: int) -> None:
+def test_simulate(d3_hexcode: HexagonalColorCode, distance: int, p: float, nr_sims: int) -> None:
```

Comment:
the distance parameter and fixture isn't really needed anymore, right? it's all covered by the code parameter.